app/src/main/java/com/github/se/gomeet/ui/mainscreens/Events.kt

```diff
@@ -175,42 +177,48 @@ fun Events(nav: NavigationActions) {
                       EventDate = eventDate,
                       ProfilePicture = R.drawable.gomeet_logo,
                       EventPicture = R.drawable.intbee_logo,
-                      Verified = true)
+                      Verified = true,
+                      nav = nav)
                   EventWidget(
-                      UserName = "EPFL Chess Club",
-                      EventName = "Chess Tournament",
+                      UserName = "EPFL",
+                      EventName = "Chess",
                       EventDate = eventDate,
                       ProfilePicture = R.drawable.gomeet_logo,
                       EventPicture = R.drawable.intbee_logo,
-                      Verified = true)
+                      Verified = true,
+                      nav = nav)
                   EventWidget(
                       UserName = "EPFL Chess Club",
                       EventName = "Chess Tournament",
                       EventDate = eventDate,
                       ProfilePicture = R.drawable.gomeet_logo,
                       EventPicture = R.drawable.intbee_logo,
-                      Verified = true)
+                      Verified = true,
+                      nav = nav)
                   EventWidget(
                       UserName = "EPFL Chess Club",
                       EventName = "Chess Tournament",
                       EventDate = eventDate,
                       ProfilePicture = R.drawable.gomeet_logo,
                       EventPicture = R.drawable.intbee_logo,
-                      Verified = true)
+                      Verified = true,
+                      nav = nav)
                   EventWidget(
                       UserName = "EPFL Chess Club",
                       EventName = "Chess Tournament",
                       EventDate = eventDate,
                       ProfilePicture = R.drawable.gomeet_logo,
                       EventPicture = R.drawable.intbee_logo,
-                      Verified = true)
+                      Verified = true,
+                      nav = nav)
                   EventWidget(
                       UserName = "EPFL Chess Club",
                       EventName = "Chess Tournament",
                       EventDate = eventDate,
                       ProfilePicture = R.drawable.gomeet_logo,
                       EventPicture = R.drawable.intbee_logo,
-                      Verified = true)
+                      Verified = true,
+                      nav = nav)
                 }
```

Comment:
I suggest leveraging the getAllEvents method from the EventViewModel class to fetch all events, rather than hardcoding them directly. This approach would enhance scalability and maintainability. For integrating the EventViewModel into our UI components, we could consider passing an instance of EventViewModel as a parameter to the Events composable. This would ensure that our UI is dynamically updated with the data managed by the ViewModel. What are your thoughts on incorporating this change?

Reply:
This will be done by Vlad. this is only a placeholder for testing the UI.